ampligraph/latent_features/models/EmbeddingModel.py

```diff
@@ -944,7 +944,7 @@ def _training_data_generator(self):
             else:
                 yield np.squeeze(out_triples), unique_entities, entity_embeddings
 
-    def fit(self, X, early_stopping=False, early_stopping_params={}, focusE_numeric_edge_values=None):
+    def fit(self, X, early_stopping=False, early_stopping_params={}, focusE_numeric_edge_values=None, tensorboard=False):
```

Comment:
I would suggest replacing this with tensorboard_logs_path =None. If none then dont save tf summary, if not save it at desired location

Reply:
Good point, will add that thanks.